src/main/kotlin/org/hyacinthbots/saffronstatus/SaffronStatus.kt

```diff
@@ -28,11 +31,12 @@ suspend fun main() {
         }
 
         extensions {
-            add(::DowntimeWatcher)
+			add(::WatchedBot)
+			add(::DowntimeNotifier)
         }
 
 		presence {
-			watching("Hyacinth Bots status'!")
+			watching("Bots for downtime!")
```

Comment:
No capital b. If you feel like it, can we make this display the number of bots it's watching?

Reply:
Would be cool, but we risk duplicates with the current db system. I could rewrite the db system first to add one instance of the bot to the db and then add guilds as "subscribers" but idk, up to you. This system works well